app/sliders/importer.py

```diff
@@ -166,6 +174,48 @@ def init_nested_dict(self):
         :return: defaultdict, a nested dictionary
         """
         return defaultdict(self.init_nested_dict)
+    
+
+class FUEDataParser:
+    def parse(self, sheets):
+        data = self.extract(sheets)
+
+        return data
+    
+    def extract(self, sheets):
+        """
+        Extract Fue Expenses data.
+
+        :param sheets: dict, where keys are names of sheets and values are pandas DataFrames
+        :return: nested dict, with the following shape {area: {year: {x: unit[], y: value[]}}}
+        """
+        extracted = self.init_nested_dict()
+        relevant_years = [2011, 2020]
+        for sheet_key in sheets:
+            basis, _, area = sheet_key.split("_")
+            if basis == "fue":
+                df = sheets[sheet_key]
+                row = df.loc[df["Jahr"].isin(relevant_years)]
+
+                for year in relevant_years:
+                    extracted[area][str(year)] = {"x": [], "y": []}
+                    for unit in mapping_fue:
```

Comment:
I would rename "unit" to "branch"

Reply:
We already have the list of branches in the config. You can access it with self.config["filters"]["single_choice_highlight"]

Reply:
cool. done